src/ops/ops.go

```diff
@@ -124,6 +125,71 @@ func (o *ops) SystemctlAction(action string, args ...string) error {
 	return errors.Wrapf(err, "Failed executing systemctl %s %s", action, args)
 }
 
+var ostreeOutputRegex = regexp.MustCompile(`Imported: (\w+)`)
+
+func (o *ops) importOSTreeCommit(liveLogger io.Writer) (string, error) {
+	ostreeReleasePullSpec := fmt.Sprintf("ostree-unverified-registry:%s", o.installerConfig.CoreosImage)
+	out, err := o.ExecPrivilegeCommand(liveLogger, "ostree", "container", "unencapsulate", "--authfile", "/root/.docker/config.json", "--quiet", "--repo", "/ostree/repo", ostreeReleasePullSpec)
```

Comment:
nit: this is probably not gonna change often (or possibly never), however wouldn't the docker config file path be better in a constant? (there's already one in installer package, should we share such constant - in a different package of course - as we are relying that this file has been extracted there?)